transfuse-core/src/main/java/org/androidtransfuse/gen/FilerResourceWriter.java

```diff
@@ -41,18 +41,13 @@ public FilerResourceWriter(Filer filer) {
     public OutputStream openBinary(JPackage pkg, String fileName) throws IOException {
         FileObject resource = filer.createResource(StandardLocation.SOURCE_OUTPUT, pkg.name(), fileName);
 
-        OutputStream os = resource.openOutputStream();
-        openStreams.add(os);
-
+        OutputStream os = new WriterOutputStream(resource.openWriter());
```

Comment:
This constructor is depreciated, we should define a charset.

Reply:
I changed it to  
`OutputStream os = new WriterOutputStream(sourceFile.openWriter(), "UTF-8"); `

is it okay to use UTF-8 ?